### DISCUSSION TITLE
键盘整体优化

### DISCUSSION THREAD

**Discussion by jokerxsun** [imfuxiao/Hamster#49] [Ideas]
首先，特别感谢大佬能够上架并且开源。辛苦，感谢。

恰好最近这段时间在鼓捣键盘，本来也想闲下来的时候自己开发一个，机缘巧合看见了大佬开源的。正如你在回答之前一些讨论的问题时候所说，大部分人其实都是开箱即用的，一顿折腾之后又会回归到最开始的状态。

其实能用 iRime 输入法的，应该都是被搜狗、系统输入法过滤之后的用户了，可能大部分是因为隐私、系统输入法联想词库垃圾而选择 iRime。如果说 iPhone 上输入法做的最好的，绝对是系统输入法了，但是系统输入法的联想、词库功能实在是不敢恭维。我感觉如果可以的话，可以先锚定好一个方向，百分百的复刻系统输入法，在复刻完成后，再去思考其他的功能，这样也能稳定的推进下去，逐步迭代。

当然，我也 Fork 了您的项目，我打算抽个时间先改一下，解决下现有bug，到时候 PR 辛苦您审核。

**Comment by Ricadre** [comment]
基本和我的需求一样了，需要一个100%复刻原生键盘的输入法（因为原生的键盘误触我觉得是最小的），同时又有rime的词库和全平台同步功能

**Reply by jokerxsun** [reply]
是的，所以我决定自己开发下。因为用原生键盘有肌肉记忆了，再换别的难受。整体来说这是最好的搭配。

**Reply by Ricadre** [reply]
期待大佬的作品

**Comment by imfuxiao** [comment]
很好的想法, 期待提交PR. 一起把这款应用做好.

**Reply by jokerxsun** [reply]
最近绩效月工作太忙了，刚好这周有时间。大佬所有的代码都提交了吗？

**Reply by imfuxiao** [reply]
还没, 今晚发1.1.4, 在做最后测试, 没有问题就提交
